### PR TITLE
Cypress/fe 1458

### PR DESCRIPTION
**Pull request by marcinstachowicz**
## Automation tests for Storybook components

**Added to testing scope:**
- Detail,
- Dropdown - not all test cases because of this component will be deprecated and replaced by Select

**Improvements:**
- move reusable locators to common locators
- move reusable steps to common steps
- minor readme update

### CODE REVIEW

File: cypress/integration/dropdown.feature [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      | <>                      |
+
+  @positive
+  Scenario Outline: Change Dropdown component field help
+    When I set field help to "<fieldHelp>"
+    Then Field help on preview is set to "<fieldHelp>"
+    Examples:
+      | fieldHelp                |
+      | Example Test             |
+      | 1!@#$%^*()_+-=~[];:.,?{} |
```
Comment (DlgSHi) [inline cypress/integration/dropdown.feature]: maybe we should use German / Spanish special characters for all input ?

File: cypress/locators/dropdown/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { FORM } from '../locators';
+import {
+  DROPDOWN_INPUT, DROPDOWN_COMPONENT, DROPDOWN_LABEL,
+} from './locators';
+
+// knobs locators
+export const readOnlyCheckbox = () => cy.get(FORM).contains('readOnly').find('input');
```
Comment (DlgSHi) [inline cypress/locators/dropdown/index.js]: should we move readOnly to common staff ?

Comment (marcinstachowicz) [reply cypress/locators/dropdown/index.js]: readOnlyCheckbox is used only for Dropdown component, in my opinion current location is correct

Comment (DlgSHi) [reply cypress/locators/dropdown/index.js]: good point

File: cypress/integration/dropdown.feature (hunk 1 above)
Comment (marcinstachowicz) [reply cypress/integration/dropdown.feature]: fixed

**Pull request merged**